test/integration/runQuery.test.ts

```diff
@@ -79,7 +79,8 @@ test("read and write methods: write options enable not waiting to return until a
 });
 
 test("read and write methods: returns transaction receipt when contract is called directly", async function (t) {
-  const provider = new ethers.providers.JsonRpcProvider();
+  const provider = getDefaultProvider("http://127.0.0.1:8545");
```

Comment:
There's a potential to use the local-tableland `getConnection` helper, but I think since the connect method is being tested here these test should do the full connection creation.

Reply:
Yeh using the SDK to test the SDK is probably best avoided ;)